modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -498,22 +518,29 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       throw new Error(`Missing BitGo to ${recipient} key share`);
     }
 
-    await this.verifyWalletSignatures(
-      userGpgKey.publicKey,
-      backupGpgKey.publicKey,
-      bitgoKeychain,
-      bitGoToRecipientShare.publicShare,
-      recipientIndex
-    );
+    // await this.verifyWalletSignatures(
```

Comment:
todo uncomment ofc after figuring out what to do

Reply:
do we have a ticket for this todo?